src/ProjectNeon/Assets/Features/Combatants/MemberState.cs

```diff
@@ -1,13 +1,15 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using UnityEngine;
 
 public sealed class MemberState : IStats
 {
     private readonly IStats _baseStats;
     private readonly List<IStats> _battleAdditiveMods = new List<IStats>();
     private readonly List<ITemporalState> _additiveMods = new List<ITemporalState>();
     private readonly List<ITemporalState> _multiplierMods = new List<ITemporalState>();
+    public FeedType FeedType { get; private set; } = FeedType.NONE;
```

Comment:
No class-specific methods allowed in `MemberState`.

Make this more generic. 

Can have an extension method that maps to/from FeedType if desired for nice call syntax on the outside.

Reply:
@SilasReinagel Extension methods can't access private members, so where would I store `FeedType` ?

Reply:
@paulodamaso No-one said anything about private. 

FeedType should either live outside this class completely, or be stored in a generic way, like in a key/value structure.

Reply:
@SilasReinagel That's what I'm talking about: it doesn't make a difference if we store `FeedType` data in a dictionary `feedType<int, String>` inside `MemberState` or in a `FeedType` field because we will be tiying the concepts anyway. And worse: to use extension methods we would have to make this field public, making we have free acces to it. I'll came up with another solution for this